# What are proteins and how do we know their structures?

**Proteins are large molecules that are essential to life as we know it. If you examine any biological process, you will find proteins involved and often playing critical roles. The structures of proteins are strongly related to their functions, but determining their 3D structures is very difficult.**

## **What is a protein?**

Each protein molecule consists of one or more chains, made by linking small molecules called amino acids. The amino acid residues are arranged in a specific sequence along each chain. These chains are folded into a distinctive 3D shape, determined by interactions between the amino acids.

There is a strong relationship between protein structure and protein function. The 3D shape of the protein determines the actions it can perform.

Figuring out protein structure is like opening a black box. Revealing the 3D structure enables scientists to determine how exactly proteins perform their duties.












##### Salmonella alpha-2-macroglobulin (PDB ID: 4U48)

Provides protection against proteases secreted from hosts

##### HIV-1 Reverse Transcriptase (PDB ID: 8U6H)

One of the key proteins responsible for the replication cycle of HIV-1 in the host.

##### SARS-CoV main protease (PDB ID: 6Y7M)

This essential coronavirus protease processes the polyproteins translated from the viral RNA

##### Circadian Clock Protein KaiA (PDB ID: 1R8J)

A tiny clockmaker that orchestrates the daily rhythms of cyanobacteria. KaiA regulates other proteins in a 24-hour cycle, like a metronome for time. To keep the rhythm on track, it senses light and other cues

Figure 1. The 3D structures depicted in this figure provide a visual journey into the functional diversity of proteins.

## **The struggle to determine protein structures**

For several decades, it has been possible to determine protein structures at atomic levels of detail, using experimental methods. Fundamental breakthroughs in the 1950s and 60s allowed the use of X-ray crystallography and later nuclear magnetic resonance (NMR). The 21st century has seen growing use of cryogenic electron microscopy (cryo-EM), especially after the “resolution revolution” ([Kühlbrandt, 2014](https://doi.org/10.1126/science.1251652)).

However, despite the progress made in experimental techniques, determining protein structure experimentally is inherently time and labour-consuming, and involves many troubleshooting steps. As a result, the number of known protein sequences has always been greater than the number of available protein structures. In January 2024, the number of experimental structures available in the PDB was slightly over 215,000, in comparison to the staggering ~250 million protein sequences available through UniProt.

![](https://ftp.ebi.ac.uk/pub/training/2024/On-demand/Proteins.gif)


Figure 2. Unlocking protein structures. Three experimental methods used
for determining protein structure: X-ray, NMR and cryo-EM. AlphaFold2, a powerful AI-driven method, has
revolutionised the field by predicting protein structures with remarkable accuracy. Source: [What Is AlphaFold? | NEJM](https://youtu.be/7q8Uw3rmXyE) , “A Holy Grail — The Prediction of Protein Structure” ([Altman, 2023](https://doi.org/10.1056/nejmcibr2307735))

Furthermore, the gap has widened considerably in recent years, thanks to advances in DNA sequencing. Experimental structure determination can’t keep up with the pace of sequencing, so there have been hundreds of millions of known proteins with unknown structures.

It is therefore extremely important to figure out the structures of these proteins with reasonable accuracy and speed, in order to narrow the gap between sequences and available structures.

AlphaFold, as explained in subsequent sections, revolutionised the field by providing an effective way to predict highly accurate protein structures.