# References

* Abanades, B., Wong, W. K., Boyles, F., Georges, G., Bujotzek, A., & Deane, C. M. (2023). ImmuneBuilder: Deep-Learning models for predicting the structures of immune proteins. Communications Biology, 6(1), Article 1. https://doi.org/10.1038/s42003-023-04927-7
* Agirre, J., Atanasova, M., Bagdonas, H., Ballard, C. B., Baslé, A., Beilsten-Edmands, J., Borges, R. J., Brown, D. G., Burgos-Mármol, J. J., Berrisford, J. M., Bond, P. S., Caballero, I., Catapano, L., Chojnowski, G., Cook, A. G., Cowtan, K. D., Croll, T. I., Debreczeni, J. É., Devenish, N. E., … Yamashita, K. (2023). The CCP4 suite: Integrative software for macromolecular crystallography. Acta Crystallographica Section D: Structural Biology, 79(6), 449–461. https://doi.org/10.1107/S2059798323003595
* Akdel, M., Pires, D. E. V., Pardo, E. P., Jänes, J., Zalevsky, A. O., Mészáros, B., Bryant, P., Good, L. L., Laskowski, R. A., Pozzati, G., Shenoy, A., Zhu, W., Kundrotas, P., Serra, V. R., Rodrigues, C. H. M., Dunham, A. S., Burke, D., Borkakoti, N., Velankar, S., … Beltrao, P. (2022). A structural biology community assessment of AlphaFold2 applications. Nature Structural & Molecular Biology, 29(11), Article 11. https://doi.org/10.1038/s41594-022-00849-w
* Alderson, T. R., Pritišanac, I., Kolarić, Đ., Moses, A. M., & Forman-Kay, J. D. (2023). Systematic identification of conditionally folded intrinsically disordered regions by AlphaFold2. Proceedings of the National Academy of Sciences, 120(44), e2304302120. https://doi.org/10.1073/pnas.2304302120
* Altman, Russ B. “A holy grail — the prediction of protein structure.” New England Journal of Medicine, vol. 389, no. 15, 12 Oct. 2023, pp. 1431–1434, https://doi.org/10.1056/nejmcibr2307735.
* Barbarin-Bocahu, I., & Graille, M. (2022). The X-ray crystallography phase problem solved thanks to AlphaFold and RoseTTAFold models: A case-study report. Acta Crystallographica Section D: Structural Biology, 78(4), 517–531. https://doi.org/10.1107/S2059798322002157
* Barrio-Hernandez, I., Yeo, J., Jänes, J., Mirdita, M., Gilchrist, C. L. M., Wein, T., Varadi, M., Velankar, S., Beltrao, P., & Steinegger, M. (2023). Clustering predicted structures at the scale of the known protein universe. Nature, 622(7983), Article 7983. https://doi.org/10.1038/s41586-023-06510-w
* Bartolec, T. K., Vázquez-Campos, X., Norman, A., Luong, C., Johnson, M., Payne, R. J., Wilkins, M. R., Mackay, J. P., & Low, J. K. K. (2023). Cross-linking mass spectrometry discovers, evaluates, and corroborates structures and protein–protein interactions in the human cell. Proceedings of the National Academy of Sciences, 120(17), e2219418120. https://doi.org/10.1073/pnas.2219418120
* Benner, S. A., & Gerloff, D. (1991). Patterns of divergence in homologous proteins as indicators of secondary and tertiary structure: A prediction of the structure of the catalytic domain of protein kinases. Advances in Enzyme Regulation, 31, 121–181. https://doi.org/10.1016/0065-2571(91)90012-B
* Bordin, N., Sillitoe, I., Nallapareddy, V., Rauer, C., Lam, S. D., Waman, V. P., Sen, N., Heinzinger, M., Littmann, M., Kim, S., Velankar, S., Steinegger, M., Rost, B., & Orengo, C. (2023). AlphaFold2 reveals commonalities and novelties in protein structure space for 21 model organisms. Communications Biology, 6(1), Article 1. https://doi.org/10.1038/s42003-023-04488-9
* Bradley, P. (2023). Structure-based prediction of T cell receptor:peptide-MHC interactions. eLife, 12, e82813. https://doi.org/10.7554/eLife.82813
* Chen, J., Fruhauf, A., Fan, C., Ponce, J., Ueberheide, B., Bhabha, G., & Ekiert, D. C. (2023). Structure of an endogenous mycobacterial MCE lipid transporter. Nature, 620(7973), Article 7973. https://doi.org/10.1038/s41586-023-06366-0
* Cheng, J., Novati, G., Pan, J., Bycroft, C., Žemgulytė, A., Applebaum, T., Pritzel, A., Wong, L. H., Zielinski, M., Sargeant, T., Schneider, R. G., Senior, A. W., Jumper, J., Hassabis, D., Kohli, P., & Avsec, Ž. (2023). Accurate proteome-wide missense variant effect prediction with AlphaMissense. Science, 381(6664), eadg7492. https://doi.org/10.1126/science.adg7492
* Chojnowski, G. (2022). Sequence-assignment validation in cryo-EM models with checkMySequence. Acta Crystallographica Section D: Structural Biology, 78(7), 806–816. https://doi.org/10.1107/S2059798322005009
* del Alamo, D., Sala, D., Mchaourab, H. S., & Meiler, J. (2022). Sampling alternative conformational states of transporters and receptors with AlphaFold2. eLife, 11, e75751. https://doi.org/10.7554/eLife.75751
* Dobson, L., Szekeres, L. I., Gerdán, C., Langó, T., Zeke, A., & Tusnády, G. E. (2023). TmAlphaFold database: Membrane localization and evaluation of AlphaFold2 predicted alpha-helical transmembrane protein structures. Nucleic Acids Research, 51(D1), D517–D522. https://doi.org/10.1093/nar/gkac928
* Durairaj, J., Waterhouse, A. M., Mets, T., Brodiazhenko, T., Abdullah, M., Studer, G., Tauriello, G., Akdel, M., Andreeva, A., Bateman, A., Tenson, T., Hauryliuk, V., Schwede, T., & Pereira, J. (2023). Uncovering new families and folds in the natural protein universe. Nature, 622(7983), Article 7983. https://doi.org/10.1038/s41586-023-06622-3
* Elofsson, A. (2023). Progress at protein structure prediction, as seen in CASP15. Current Opinion in Structural Biology, 80, 102594. https://doi.org/10.1016/j.sbi.2023.102594
* Ennist, N. M., Stayrook, S. E., Dutton, P. L., & Moser, C. C. (2022). Rational design of photosynthetic reaction center protein maquettes. Frontiers in Molecular Biosciences, 9. https://www.frontiersin.org/articles/10.3389/fmolb.2022.997295
* Erickson, E., Gado, J. E., Avilán, L., Bratti, F., Brizendine, R. K., Cox, P. A., Gill, R., Graham, R., Kim, D.-J., König, G., Michener, W. E., Poudel, S., Ramirez, K. J., Shakespeare, T. J., Zahn, M., Boyd, E. S., Payne, C. M., DuBois, J. L., Pickford, A. R., … McGeehan, J. E. (2022). Sourcing thermotolerant poly(ethylene terephthalate) hydrolase scaffolds from natural diversity. Nature Communications, 13(1), Article 1. https://doi.org/10.1038/s41467-022-35237-x
* Evans, R., O’Neill, M., Pritzel, A., Antropova, N., Senior, A., Green, T., Žídek, A., Bates, R., Blackwell, S., Yim, J., Ronneberger, O., Bodenstein, S., Zielinski, M., Bridgland, A., Potapenko, A., Cowie, A., Tunyasuvunakool, K., Jain, R., Clancy, E., … Hassabis, D. (2022). Protein complex prediction with AlphaFold-Multimer (p. 2021.10.04.463034). bioRxiv. https://doi.org/10.1101/2021.10.04.463034
* Fowler, N. J., & Williamson, M. P. (2022). The accuracy of protein structures in solution determined by AlphaFold and NMR. Structure, 30(7), 925-933.e2. https://doi.org/10.1016/j.str.2022.04.005
* Gan, Z. Y., Callegari, S., Cobbold, S. A., Cotton, T. R., Mlodzianoski, M. J., Schubert, A. F., Geoghegan, N. D., Rogers, K. L., Leis, A., Dewson, G., Glukhova, A., & Komander, D. (2022). Activation mechanism of PINK1. Nature, 602(7896), Article 7896. https://doi.org/10.1038/s41586-021-04340-2
* Giri, N., Roy, R. S., & Cheng, J. (2023). Deep learning for reconstructing protein structures from cryo-EM density maps: Recent advances and future directions. Current Opinion in Structural Biology, 79, 102536. https://doi.org/10.1016/j.sbi.2023.102536
* Göbel, U., Sander, C., Schneider, R., & Valencia, A. (1994). Correlated mutations and residue contacts in proteins. Proteins: Structure, Function, and Bioinformatics, 18(4), 309–317. https://doi.org/10.1002/prot.340180402
* Guo, H.-B., Perminov, A., Bekele, S., Kedziora, G., Farajollahi, S., Varaljay, V., Hinkle, K., Molinero, V., Meister, K., Hung, C., Dennis, P., Kelley-Loughnane, N., & Berry, R. (2022). AlphaFold2 models indicate that protein sequence determines both structure and dynamics. Scientific Reports, 12(1), Article 1. https://doi.org/10.1038/s41598-022-14382-9
* Healy, M. D., McNally, K. E., Butkovič, R., Chilton, M., Kato, K., Sacharz, J., McConville, C., Moody, E. R. R., Shaw, S., Planelles-Herrero, V. J., Yadav, S. K. N., Ross, J., Borucu, U., Palmer, C. S., Chen, K.-E., Croll, T. I., Hall, R. J., Caruana, N. J., Ghai, R., … Cullen, P. J. (2023). Structure of the endosomal Commander complex linked to Ritscher-Schinzel syndrome. Cell, 186(10), 2219-2237.e29. https://doi.org/10.1016/j.cell.2023.04.003
* Hekkelman, M. L., de Vries, I., Joosten, R. P., & Perrakis, A. (2023). AlphaFill: Enriching AlphaFold models with ligands and cofactors. Nature Methods, 20(2), Article 2. https://doi.org/10.1038/s41592-022-01685-y
* Humphreys, I. R., Pei, J., Baek, M., Krishnakumar, A., Anishchenko, I., Ovchinnikov, S., Zhang, J., Ness, T. J., Banjade, S., Bagde, S. R., Stancheva, V. G., Li, X.-H., Liu, K., Zheng, Z., Barrero, D. J., Roy, U., Kuper, J., Fernández, I. S., Szakal, B., … Baker, D. (2021). Computed structures of core eukaryotic protein complexes. Science, 374(6573), eabm4805. https://doi.org/10.1126/science.abm4805
* Jumper, J., Evans, R., Pritzel, A., Green, T., Figurnov, M., Ronneberger, O., Tunyasuvunakool, K., Bates, R., Žídek, A., Potapenko, A., Bridgland, A., Meyer, C., Kohl, S. A. A., Ballard, A. J., Cowie, A., Romera-Paredes, B., Nikolov, S., Jain, R., Adler, J., … Hassabis, D. (2021). Highly accurate protein structure prediction with AlphaFold. Nature, 596(7873), Article 7873. https://doi.org/10.1038/s41586-021-03819-2
* Keegan, R. M., McNicholas, S. J., Thomas, J. M. H., Simpkin, A. J., Simkovic, F., Uski, V., Ballard, C. C., Winn, M. D., Wilson, K. S., & Rigden, D. J. (2018). Recent developments in MrBUMP: Better search-model preparation, graphical interaction with search models, and solution improvement and assessment. Acta Crystallographica Section D: Structural Biology, 74(3), 167–182. https://doi.org/10.1107/S2059798318003455
* Korber, B. T., Farber, R. M., Wolpert, D. H., & Lapedes, A. S. (1993). Covariation of mutations in the V3 loop of human immunodeficiency virus type 1 envelope protein: An information theoretic analysis. Proceedings of the National Academy of Sciences of the United States of America, 90(15), 7176–7180.
* Kreitz, J., Friedrich, M. J., Guru, A., Lash, B., Saito, M., Macrae, R. K., & Zhang, F. (2023). Programmable protein delivery with a bacterial contractile injection system. Nature, 616(7956), Article 7956. https://doi.org/10.1038/s41586-023-05870-7.
* Kryshtafovych, A., Schwede, T., Topf, M., Fidelis, K., & Moult, J. (2023). Critical assessment of methods of protein structure prediction (casp)—round xv. Proteins: Structure, Function, and Bioinformatics, 91(12), 1539–1549. https://doi.org/10.1002/prot.26617
* Kühlbrandt, W. (2014). The Resolution Revolution. Science, 343(6178), 1443–1444. https://doi.org/10.1126/science.1251652
* Lim, Y., Tamayo-Orrego, L., Schmid, E., Tarnauskaite, Z., Kochenova, O. V., Gruar, R., Muramatsu, S., Lynch, L., Schlie, A. V., Carroll, P. L., Chistol, G., Reijns, M. A. M., Kanemaki, M. T., Jackson, A. P., & Walter, J. C. (2023). In silico protein interaction screening uncovers DONSON’s role in replication initiation. Science, 381(6664), eadi3448. https://doi.org/10.1126/science.adi3448
* Lomize, A. L., Schnitzer, K. A., Todd, S. C., Cherepanov, S., Outeiral, C., Deane, C. M., & Pogozheva, I. D. (2022). Membranome 3.0: Database of single-pass membrane proteins with AlphaFold models. Protein Science, 31(5), e4318. https://doi.org/10.1002/pro.4318
* Mariani, V., Biasini, M., Barbato, A., & Schwede, T. (2013). lDDT: A local superposition-free score for comparing protein structures and models using distance difference tests. Bioinformatics, 29(21), 2722–2728. https://doi.org/10.1093/bioinformatics/btt473
* McCafferty, C. L., Pennington, E. L., Papoulas, O., Taylor, D. W., & Marcotte, E. M. (2023). Does AlphaFold2 model proteins’ intracellular conformations? An experimental test using cross-linking mass spectrometry of endogenous ciliary proteins. Communications Biology, 6(1), Article 1. https://doi.org/10.1038/s42003-023-04773-7
* McCoy, A. J., Sammito, M. D., & Read, R. J. (2022). Implications of AlphaFold2 for crystallographic phasing by molecular replacement. Acta Crystallographica Section D: Structural Biology, 78(1), 1–13. https://doi.org/10.1107/S2059798321012122
* Mikhaylov, V., & Levine, A. J. (2023). Accurate modeling of peptide-MHC structures with AlphaFold (p. 2023.03.06.531396). bioRxiv. https://doi.org/10.1101/2023.03.06.531396
* Millán, C., Keegan, R. M., Pereira, J., Sammito, M. D., Simpkin, A. J., McCoy, A. J., Lupas, A. N., Hartmann, M. D., Rigden, D. J., & Read, R. J. (2021). Assessing the utility of CASP14 models for molecular replacement. Proteins: Structure, Function, and Bioinformatics, 89(12), 1752–1769. https://doi.org/10.1002/prot.26214
* Mirdita, M., Schütze, K., Moriwaki, Y., Heo, L., Ovchinnikov, S., & Steinegger, M. (2022). ColabFold: Making protein folding accessible to all. Nature Methods, 19(6), Article 6. https://doi.org/10.1038/s41592-022-01488-1
* Monzon, V., Paysan-Lafosse, T., Wood, V., & Bateman, A. (2022). Reciprocal best structure hits: Using AlphaFold models to discover distant homologues. Bioinformatics Advances, 2(1), vbac072. https://doi.org/10.1093/bioadv/vbac072
* Mosalaganti, S., Obarska-Kosinska, A., Siggel, M., Taniguchi, R., Turoňová, B., Zimmerli, C. E., Buczak, K., Schmidt, F. H., Margiotta, E., Mackmull, M.-T., Hagen, W. J. H., Hummer, G., Kosinski, J., & Beck, M. (2022). AI-based structure prediction empowers integrative structural analysis of human nuclear pores. Science, 376(6598), eabm9506. https://doi.org/10.1126/science.abm9506
* Oeffner, R. D., Croll, T. I., Millán, C., Poon, B. K., Schlicksup, C. J., Read, R. J., & Terwilliger, T. C. (2022). Putting AlphaFold models to work with phenix.process\_predicted\_model and ISOLDE. Acta Crystallographica Section D: Structural Biology, 78(11), 1303–1314. https://doi.org/10.1107/S2059798322010026
* Pei, J., & Cong, Q. (2023). AFTM: A database of transmembrane regions in the human proteome predicted by AlphaFold. Database, 2023, baad008. https://doi.org/10.1093/database/baad008
* Peng, C., Wang, Z., Zhao, P., Ge, W., & Huang, C. (2023). AbFold—An AlphaFold Based Transfer Learning Model for Accurate Antibody Structure Prediction (p. 2023.04.20.537598). bioRxiv. https://doi.org/10.1101/2023.04.20.537598
* Piovesan, D., Monzon, A. M., & Tosatto, S. C. E. (2022). Intrinsic protein disorder and conditional folding in AlphaFoldDB. Protein Science, 31(11), e4466. https://doi.org/10.1002/pro.4466
* Ruffolo, J. A., Chu, L.-S., Mahajan, S. P., & Gray, J. J. (2023). Fast, accurate antibody structure prediction from deep learning on massive set of natural antibodies. Nature Communications, 14(1), Article 1. https://doi.org/10.1038/s41467-023-38063-x
* Simpkin, A. J., Caballero, I., McNicholas, S., Stevenson, K., Jiménez, E., Sánchez Rodríguez, F., Fando, M., Uski, V., Ballard, C., Chojnowski, G., Lebedev, A., Krissinel, E., Usón, I., Rigden, D. J., & Keegan, R. M. (2023). Predicted models and CCP4. Acta Crystallographica Section D: Structural Biology, 79(9), 806–819. https://doi.org/10.1107/S2059798323006289
* Stahl, K., Graziadei, A., Dau, T., Brock, O., & Rappsilber, J. (2023). Protein structure prediction with in-cell photo-crosslinking mass spectrometry and deep learning. Nature Biotechnology, 41(12), Article 12. https://doi.org/10.1038/s41587-023-01704-z
* Taylor, W. R., & Hatrick, K. (1994). Compensating changes in protein multiple sequence alignments. Protein Engineering, Design and Selection, 7(3), 341–348. https://doi.org/10.1093/protein/7.3.341
* Tejero, R., Huang, Y. J., Ramelot, T. A., & Montelione, G. T. (2022). AlphaFold Models of Small Proteins Rival the Accuracy of Solution NMR Structures. Frontiers in Molecular Biosciences, 9. https://www.frontiersin.org/articles/10.3389/fmolb.2022.877000
* Terwilliger, T. C., Liebschner, D., Croll, T. I., Williams, C. J., McCoy, A. J., Poon, B. K., Afonine, P. V., Oeffner, R. D., Richardson, J. S., Read, R. J., & Adams, P. D. (2023). AlphaFold predictions are valuable hypotheses and accelerate but do not replace experimental structure determination. Nature Methods, 1–7. https://doi.org/10.1038/s41592-023-02087-4
* Tunyasuvunakool, K., Adler, J., Wu, Z., Green, T., Zielinski, M., Žídek, A., Bridgland, A., Cowie, A., Meyer, C., Laydon, A., Velankar, S., Kleywegt, G. J., Bateman, A., Evans, R., Pritzel, A., Figurnov, M., Ronneberger, O., Bates, R., Kohl, S. A. A., … Hassabis, D. (2021). Highly accurate protein structure prediction for the human proteome. Nature, 596(7873), Article 7873. https://doi.org/10.1038/s41586-021-03828-1
* van Kempen, M., Kim, S. S., Tumescheit, C., Mirdita, M., Lee, J., Gilchrist, C. L. M., Söding, J., & Steinegger, M. (2023). Fast and accurate protein structure search with Foldseek. Nature Biotechnology, 1–4. https://doi.org/10.1038/s41587-023-01773-0
* Wayment-Steele, H. K., Ojoawo, A., Otten, R., Apitz, J. M., Pitsawong, W., Hömberger, M., Ovchinnikov, S., Colwell, L., & Kern, D. (2023). Predicting multiple conformations via sequence clustering and AlphaFold2. Nature, 1–8. https://doi.org/10.1038/s41586-023-06832-9
* Weeratunga, S., Gormal, R. S., Liu, M., Eldershaw, D., Livingstone, E. K., Malapaka, A., Wallis, T. P., Bademosi, A. T., Jiang, A., Healy, M. D., Meunier, F. A., & Collins, B. M. (2023). Interrogation and validation of the interactome of neuronal Munc18-interacting Mint proteins with AlphaFold2 (p. 2023.02.20.529329). bioRxiv. https://doi.org/10.1101/2023.02.20.529329
* Wicky, B. I. M., Milles, L. F., Courbet, A., Ragotte, R. J., Dauparas, J., Kinfu, E., Tipps, S., Kibler, R. D., Baek, M., DiMaio, F., Li, X., Carter, L., Kang, A., Nguyen, H., Bera, A. K., & Baker, D. (2022). Hallucinating symmetric protein assemblies. Science, 378(6615), 56–61. https://doi.org/10.1126/science.add1964
* Xu, J., & Zhang, Y. (2010). How significant is a protein structure similarity with TM-score = 0.5? Bioinformatics, 26(7), 889–895. https://doi.org/10.1093/bioinformatics/btq066
* Yin, R., Ribeiro-Filho, H. V., Lin, V., Gowthaman, R., Cheung, M., & Pierce, B. G. (2023). TCRmodel2: High-resolution modeling of T cell receptor recognition using deep learning. Nucleic Acids Research, 51(W1), W569–W576. https://doi.org/10.1093/nar/gkad356
* Zeng, Y. C., Sobti, M., Quinn, A., Smith, N. J., Brown, S. H. J., Vandenberg, J. I., Ryan, R. M., O’Mara, M. L., & Stewart, A. G. (2023). Structural basis of promiscuous substrate transport by Organic Cation Transporter 1. Nature Communications, 14(1), Article 1. https://doi.org/10.1038/s41467-023-42086-9